src/ColorEditor.tsx

```diff
@@ -26,7 +26,7 @@ export function ColorEditor(props: any) {
 
   return (
     <div>
-      <Input type="text" value={props.value || 'Pick Color'} prefix={prefix} suffix={suffix} />
+      <Input type="text" value={props.value || 'Pick Color'} prefix={prefix} suffix={suffix} readOnly={true} />
```

Comment:
Added readonly to silence a react error. this is a read only field (hidden) nevertheless.